### PR TITLE
fix broken links to externally defined terms

### PR DESCRIPTION
**Pull request by pchampin**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/179.html" title="Last updated on Oct 16, 2023, 4:30 PM UTC (0b126a2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/179/e80c1dd...0b126a2.html" title="Last updated on Oct 16, 2023, 4:30 PM UTC (0b126a2)">Diff</a>

### CODE REVIEW

**Comment by pchampin** [comment]
NB: links to @msporny's and @dlongley's homepages are also broken

**Review by dlongley — APPROVED** [review]
Can use `https://github.com/dlongley` as a non-broken link for me.

**Review by gkellogg — APPROVED** [review]
Thanks for catching the citation errors

File: spec/index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
     // only "name" is required
     formerEditors:  [
       { name:       "Manu Sporny",
-        url:        "http://manu.sporny.org/",
+        url:        "https://github.com/msporny",
```
Comment (msporny) [inline spec/index.html:45]:
```suggestion
        url:        "https://www.linkedin.com/in/manusporny/",
```

Comment (gkellogg) [reply spec/index.html:45]: Sorry, pulled the trigger too quickly; can you do a new PR?

**Comment by msporny** [comment]
Fixed in https://github.com/w3c/rdf-canon/commit/0503facfaa0825686afc1f533f487816de54d9b7.

File: spec/index.html (hunk 1 above)
Comment (msporny) [reply spec/index.html:45]: Fixed in https://github.com/w3c/rdf-canon/commit/0503facfaa0825686afc1f533f487816de54d9b7.

**Pull request merged**